library/alloc/src/vec/mod.rs

```diff
@@ -1835,12 +1837,15 @@ impl<T, A: Allocator> Vec<T, A> {
         self.len() == 0
     }
 
-    /// Splits the collection into two at the given index.
+    /// Splits the collection in two at the given index.
     ///
     /// Returns a newly allocated vector containing the elements in the range
     /// `[at, len)`. After the call, the original vector will be left containing
     /// the elements `[0, at)` with its previous capacity unchanged.
     ///
+    /// If you do not need to own the tail `Vec`, use [`Vec::drain`] to avoid
+    /// an extra allocation.
```

Comment:
This is not technically correct, with `drain` you own the tail items, they just aren't stored in a new `Vec`. What about something like "If you don't need the tail to be stored in a separate `Vec` you can use `Vec::drain` to iterate through it, taking ownership of each element."